src/stack_aggregator.py

```diff
@@ -518,7 +522,7 @@ def get_tr_dependency_data(epv_set):
         result = execute_gremlin_dsl(url=GREMLIN_SERVER_URL_REST, payload=payload)
         if result:
             tr_epv_list['result']['data'] += result['result']['data']
-
+    logger.info('Generated Transitive Data \n\n EPV : {tr}'.format(tr=tr_list))
```

Comment:
I guess printing the entire transitive list would be too exhaustive, isn't it?

Reply:
Okay. Removing